node/src/accountant/payable_dao.rs

```diff
@@ -530,50 +654,89 @@ mod tests {
         let boxed_conn = DbInitializerReal::default()
             .initialize(&home_dir, DbInitializationConfig::test_default())
             .unwrap();
         {
-            insert_record_fn(&*boxed_conn, &wallet.to_string(), 5000, 150_000_000, None);
+            let insert = "insert into payable (wallet_address, balance_high_b, balance_low_b, \
+             last_paid_timestamp) values (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?)";
+            let mut stm = boxed_conn.prepare(insert).unwrap();
+            let params = [
+                [&wallet_0 as &dyn ToSql, &12345, &1, &45678],
+                [&wallet_1, &0, &i64::MAX, &150_000_000],
+                [&wallet_2, &3, &0, &151_000_000],
+            ]
+            .into_iter()
+            .flatten()
+            .collect::<Vec<&dyn ToSql>>();
+            stm.execute(params.as_slice()).unwrap();
         }
         let subject = PayableDaoReal::new(boxed_conn);
-        let before_account_status = subject.account_status(&wallet).unwrap();
 
         subject
-            .mark_pending_payable_rowid(&wallet, pending_payable_rowid)
+            .mark_pending_payables_rowids(&[
+                (&wallet_1, pending_payable_rowid_1),
+                (&wallet_2, pending_payable_rowid_2),
+            ])
             .unwrap();
 
-        let before_expected_status = PayableAccount {
-            wallet: wallet.clone(),
-            balance_wei: 5000,
-            last_paid_timestamp: from_time_t(150_000_000),
-            pending_payable_opt: None,
-        };
-        assert_eq!(before_account_status, before_expected_status);
-        let after_account_status = subject.account_status(&wallet).unwrap();
-        let mut after_expected_status = before_expected_status;
-        after_expected_status.pending_payable_opt = Some(PendingPayableId {
-            rowid: pending_payable_rowid,
-            hash: H256::from_uint(&U256::from(0)), //garbage
-        });
-        assert_eq!(after_account_status, after_expected_status)
+        let account_statuses = [&wallet_0, &wallet_1, &wallet_2]
+            .iter()
+            .map(|wallet| subject.account_status(wallet).unwrap())
+            .collect::<Vec<PayableAccount>>();
+        assert_eq!(
+            account_statuses,
+            vec![
+                PayableAccount {
+                    wallet: wallet_0,
+                    balance_wei: u128::try_from(BigIntDivider::reconstitute(12345, 1)).unwrap(),
+                    last_paid_timestamp: from_time_t(45678),
+                    pending_payable_opt: None,
+                },
+                PayableAccount {
+                    wallet: wallet_1,
+                    balance_wei: u128::try_from(BigIntDivider::reconstitute(0, i64::MAX)).unwrap(),
+                    last_paid_timestamp: from_time_t(150_000_000),
+                    pending_payable_opt: Some(PendingPayableId::new(
+                        pending_payable_rowid_1,
+                        Default::default()
```

Comment:
Default::default() isn't obvious if you don't have an IDE adding parameter names and/or types for you.

--or if you're not the author.